html-relative-links/projects/projects.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Projects</title>
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li>
+      <a href="../index.html">Home Page</a>
+    </li>
+    <li>
+      <a href="../about-me/about-me.html">About Me</a>
+    </li>
+    <li>
+      <a href="../projects/projects.html">Projects</a>
```

Comment:
For this line, you are exiting the folder just to go right back into it. Instead, you can just state the name of the file directly, since you are already in the directory of the file you are navigating to.